### DISCUSSION TITLE
Github Actions Disabled on my account

### DISCUSSION THREAD

**Discussion by venopyX** [community/community#65240] [Actions]
### Select Topic Area

Question

### Body

Why my github actions disabled?
Help please 😌

**Comment by seyLu** [comment]
### I checked one of your repos, when github actions started failing, and found this:

![image](https://github.com/community/community/assets/98249191/19a7c215-072d-4df2-a4ed-ceb041da50fa)

### As far as chatgpt is concerned, this meant `running a scheduled workflow every 30 minutes`.

![image](https://github.com/community/community/assets/98249191/2f975fa4-3e61-4d0b-a53c-87584fe618c2)

### I also confirmed it via actions tab.

![image](https://github.com/community/community/assets/98249191/6d3260de-78ce-4e86-91e5-6c314391ff8e)

### Yup, you should contact github support for this.

**Reply by venopyX** [reply]
I see. Thank you... Let me ask support team.
